src/Entity/Midata/GroupType.php

```diff
@@ -51,6 +52,28 @@ class GroupType
      */
     private $groupType;
 
+
+    /**
+     * Returns whether $other is the child department of this
+     * @param GroupType $other
+     * @return bool
+     */
+    public function isDepartmentParent(GroupType &$other): bool
+    {
+        try {
+            $index = array_search($this->getGroupType(), self::DEPARTMENT_HIERARCHY);
+            if (!$index) {
+                return false;
+            }
+
+            $childGroupType = self::DEPARTMENT_HIERARCHY[$index + 1];
```

Comment:
I'm not sure, but I think this behaviour is not guaranteed. As far as I know it could also be the case that a canton has departments directly as child. I'm also not sure if it is possible that departments have other departments as child.